core/sparqlbuilder/src/main/java/org/eclipse/rdf4j/sparqlbuilder/constraint/Values.java

```diff
@@ -0,0 +1,217 @@
+/* *****************************************************************************
+ * Copyright (c) 2015 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
Could you update this to the current copyright template?

You can find it here: https://github.com/eclipse/rdf4j/blob/main/CONTRIBUTING.md